src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -488,14 +489,16 @@ private List<File> getFilesToCheck() {
             // oops, we've got an additional one to process, don't
             // forget it. No sweat, it's fully resolved via the setter.
             log("Adding standalone file for audit", Project.MSG_VERBOSE);
-            allFiles.add(new File(fileName));
+            allFiles.add(Path.of(fileName).toFile());
```

Comment:
ideally to change to  `private List<Path> getFilesToCheck() {`

but it will require API change to new type for main process method of checkstyle `numErrs = rootModule.process(files);`

Reply:
we will think on it at https://github.com/checkstyle/checkstyle/issues/16857